CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unversioned
+### Added
+- #1204 - Please add a search bar to the applications overview to filter the
+  of applications
```

Comment:
You forgot "list" (`to filter the list of applications`). Also, i'm afraid the newline here kinda messes with the markdown syntax:

![screen shot 2015-07-20 at 09 49 03](https://cloud.githubusercontent.com/assets/1078545/8771669/9503fb32-2ec4-11e5-9dd0-e8738bbc7a07.png)

What if we added an ellipsis to the issue so that it fits, ie:

`- #1204 - Please add a search bar to the applications overview to filter the ...`

Just a thought ;)